umm-spec-lib/test/cmr/umm_spec/test/generate_and_parse.clj

```diff
@@ -105,7 +105,13 @@
       (doseq [target-format tested-collection-formats
               :when (not @failed-atom)
               :let [expected (expected-conversion/convert umm target-format)
+                    ;;Can't compare NumberOfInstruments because the original umm might not have it.
+                    ;;but after the round trip for iso-smap, it might be added.
+                    expected (update-in-each expected [:Platforms] update-in-each [:Instruments] assoc
```

Comment:
why would # of instruments get added in translation only sometimes? Wouldn't it be the # of child instruments?